.rubocop.yml

```diff
@@ -103,7 +103,10 @@ Naming/HeredocDelimiterNaming:
   Enabled: false
 
 Naming/UncommunicativeMethodParamName:
-  Enabled: false
+  Enabled: true
+  AllowedNames:
+    - as
+    - e
```

Comment:
`e`? where do we use it? Sounds weird

Reply:
Exception :-)

Reply:
But we still have a few of them:
<img width="499" alt="screen shot 2018-03-16 at 20 02 01" src="https://user-images.githubusercontent.com/15484/37548190-f5f77abc-2954-11e8-9e6b-d848c41aeb06.png">